pages/blog/2022Q1-summary.md

```diff
@@ -0,0 +1,175 @@
+
+2.4.0 was released, but this doesn't block us from working on something much bigger, 3.0.0 version of the specification :rocket:
+
+If you do not know yet about it, I highly recommend you to have a look at [this article](https://www.asyncapi.com/blog/async-api-spec-3.0-release) from [Harsh Mishra](https://www.linkedin.com/in/harshcasper/).
+
+Please join these efforts, as there is much work ahead of us, but not many hands doing actual work. We meet every two weeks to sync the work efforts and discuss important topics. Feel free to join :pray:
+
+I also recommend following [Sergio Moya](https://twitter.com/smoyac) and reaching for [his regular updates on specification-related efforts](https://gist.github.com/smoya)
+
+## Contributors onboarding initiatives
+
+One of [goals for community-building efforts for 2022](https://github.com/asyncapi/community/discussions/193) was to involve AsyncAPI Initiative as a mentoring organization in as many coding (but not only) events. We have many maintainers here who like to help first-time contributors land their first pull requests in an open and relaxed environment.
+
+### Google Summer of Code
+
+We did not start well. We were not accepted for [Google Summer of Code](https://summerofcode.withgoogle.com). Don't ask me why. I really do not know. I wish I could have a clear answer from them about what failed on our end :sad:
+
+We never give up here at AsyncAPI, though :smiley:
+
+### OpenForce
+
+March was a month where we joined [OpenForce](https://www.asyncapi.com/blog/openforce-2022) thanks to amazing support from [Hargun Kaur](https://www.linkedin.com/in/hkaur008/). As a result, we managed to onboard a few new folks and gained important contributions in exchange. There are actually a few PRs still open, but for now, completed are:
+- [Namya LG](https://github.com/Namyalg) introduced a great improvement to our CI by creating dedicated workflows to [search for broken links](https://github.com/asyncapi/.github/issues/92) in our documentation across all repositories.
+- [Harsh Mishra](https://www.linkedin.com/in/harshcasper/) introduced a workflow in our CI to [validate docker images](https://github.com/asyncapi/server-api/issues/50) in projects where we publish them
+- [Abhijeet Jejurkar](https://github.com/abhijeetjejurkar) helped [improving Kubernetes deployment](https://github.com/asyncapi/server-api/issues/57) for our `server-api` service.
+- [Samriddhi](https://github.com/Samridhi-98) contributed [new `/bundle` endpoint](https://github.com/asyncapi/server-api/issues/55) to our `server-api` service.
+- [Ritik Rawal](https://github.com/ritik307) contributed [new `/diff` endpoint](https://github.com/asyncapi/server-api/issues/56) to our `server-api` service.
+- [Everly Precia Suresh](https://github.com/everly-gif) added a possibility to [host API reference docs](https://github.com/asyncapi/server-api/issues/43) for our `server-api` service through a dedicated endpoint.
```

Comment:
```suggestion
- [Samriddhi](https://github.com/Samridhi-98) contributed a [new `/bundle` endpoint to our `server-api` service](https://github.com/asyncapi/server-api/issues/55).
- [Ritik Rawal](https://github.com/ritik307) contributed a [new `/diff` endpoint to our `server-api` service](https://github.com/asyncapi/server-api/issues/56).
- [Everly Precia Suresh](https://github.com/everly-gif) added a possibility to [host API reference docs for our `server-api` service](https://github.com/asyncapi/server-api/issues/43) through a dedicated endpoint.
```
**guess** 😹

Reply:
I approve the change, as the intention is good, but I have doubts if `[new /bundle endpoint to our server-api service]` is better than `[new /bundle endpoint]`. We focus on accessibility for screen readers but forget how unreadable the content becomes for non-screen readers (almost entire text becomes a highlighted text, not good experience when reading content). I think sometimes we need to balance things